### ISSUE TITLE
Dendrogram highlights error

### ISSUE THREAD

**Issue by anupriyatripathi** [biocore/gneiss#143]
```
qiime gneiss dendrogram-heatmap --i-table /Users/priya/Desktop/Knight-Dorrestein/palmspaper/palmspaper_targeted_sampleids_matched.composition.biom.qza --i-tree /Users/priya/Desktop/Knight-Dorrestein/palmspaper/palmspaper_targeted_sampleids_matched.composition.nwk.qza --m-metadata-file /Users/priya/Desktop/Knight-Dorrestein/palmspaper/palmspaper_md_sampleids_matched.txt --m-metadata-category COUNTRY --o-visualization /Users/priya/Desktop/Knight-Dorrestein/palmspaper/palmspaper_targeted_sampleids_matched.heatmap --verbose --p-ndim 4
Traceback (most recent call last):
  File "/Users/priya/anaconda2/envs/gneiss/lib/python3.5/site-packages/q2cli-2017.2.0-py3.5.egg/q2cli/commands.py", line 217, in __call__
    results = action(**arguments)
  File "<decorator-gen-136>", line 2, in dendrogram_heatmap
  File "/Users/priya/anaconda2/envs/gneiss/lib/python3.5/site-packages/qiime2-2017.2.0-py3.5.egg/qiime2/sdk/action.py", line 171, in callable_wrapper
    output_types, provenance)
  File "/Users/priya/anaconda2/envs/gneiss/lib/python3.5/site-packages/qiime2-2017.2.0-py3.5.egg/qiime2/sdk/action.py", line 305, in _callable_executor_
    ret_val = callable(output_dir=temp_dir, **view_args)
  File "/Users/priya/anaconda2/envs/gneiss/lib/python3.5/site-packages/gneiss-0.3.2-py3.5.egg/gneiss/plot/_plot.py", line 364, in dendrogram_heatmap
    highlight_width=0.01, figsize=(12, 8))
  File "/Users/priya/anaconda2/envs/gneiss/lib/python3.5/site-packages/gneiss-0.3.2-py3.5.egg/gneiss/plot/_heatmap.py", line 112, in heatmap
    _plot_highlights_dendrogram(ax_highlights, table, t, highlights)
  File "/Users/priya/anaconda2/envs/gneiss/lib/python3.5/site-packages/gneiss-0.3.2-py3.5.egg/gneiss/plot/_heatmap.py", line 155, in _plot_highlights_dendrogram
    k, l, r = node._k, node._l, node._r
AttributeError: 'SquareDendrogram' object has no attribute '_l'

Plugin error from gneiss:

  'SquareDendrogram' object has no attribute '_l'

See above for debug info.
```

**Comment by mortonjt** [comment]
@anupriyatripathi could you send over the files you used to generate this error?

**Comment by mortonjt** [comment]
This has been fixed by #171 Thanks @anupriyatripathi !
